engine/src/flutter/lib/ui/platform_dispatcher.dart

```diff
@@ -939,10 +939,12 @@ class PlatformDispatcher {
     call `updateSemantics`.
   ''')
   void updateSemantics(SemanticsUpdate update) =>
```

Comment:
you also need to update web counter part

Reply:
Let's punt this if web builds and passes tests. I opened https://github.com/flutter/flutter/issues/165219 to track this work for the web.

Reply:
Apologies, my last message is incorrect. Semantics update already works on Flutter Web:

Each `FlutterView` on web has its own semantics tree: https://github.com/flutter/flutter/blob/d261411b4c3bfb8591f352e3842f64005ba75505/engine/src/flutter/lib/web_ui/lib/src/engine/window.dart#L156

And `FlutterView.updateSemantics` updates that semantics tree: https://github.com/flutter/flutter/blob/d261411b4c3bfb8591f352e3842f64005ba75505/engine/src/flutter/lib/web_ui/lib/src/engine/window.dart#L129

I'll go ahead and close this since this is addressed :)